vendor.conf

```diff
@@ -2,7 +2,7 @@ github.com/blang/semver v3.1.0
 github.com/boltdb/bolt e9cf4fae01b5a8ff89d0ec6b32f0d9c9f79aefdd
 github.com/BurntSushi/toml a368813c5e648fee92e5f6c30e3944ff9d5e8895
 github.com/containerd/cgroups f7dd103d3e4e696aa67152f6b4ddd1779a3455a9
-github.com/containerd/containerd v1.0.0-beta.3
+github.com/containerd/containerd 8e09b565a70f534693deaecde136d5d6697df33e
```

Comment:
When you change containerd's version you also need to change it over here:

https://github.com/kubernetes-incubator/cri-containerd/blob/master/hack/versions#L3

Reply:
ok

Reply:
done